ui/app/styles/core/field.scss

```diff
@@ -67,20 +67,6 @@
     flex-shrink: 0;
     width: 100%;
   }
-  .checkbox {
```

Comment:
we never took advantage of something like .checkbox.is-primary.  In fact we don't use checkbox as a class at all.